### DISCUSSION TITLE
Shell Integration and 'context menu'

### DISCUSSION THREAD

**Discussion by Susja** [WinMerge/winmerge#621] [General]
Hello.
I don't have permission to install WinMerge on my PC hence I installed on remote share.
I created a shortcut on my Desktop pointing to the share and invoke it when I want to compare files.
The disadvantage is that 'Shell Integration' and 'context menu' does not work for me.
Hence my question: is it expected that invoking WinMerge from share is not supposed to provide 'context menu' or I could trick it somehow?
Thanks
![Capture](https://user-images.githubusercontent.com/26395390/107535357-a902e780-6b8e-11eb-8c9e-65385b94b326.PNG)

**Comment by sdottaka** [comment]
First, you seem to be using an older version of WinMerge. In older versions, you must have administrator privileges to use Shell Extension.
Newer versions do not necessarily require administrator privileges.

If you do not have administrator privileges, there are the following Per-User installers that do not require administrator privileges.

https://sourceforge.net/projects/winmerge/files/stable/2.16.10/WinMerge-2.16.10-x64-PerUser-Setup.exe/download

**Comment by Susja** [comment]
I am all set now
Thanks a lot
